bundles/org.openhab.binding.omnikinverter/src/main/java/org/openhab/binding/omnikinverter/internal/handler/OmnikInverterHandler.java

```diff
@@ -58,11 +59,9 @@ public OmnikInverterHandler(Thing thing) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (OmnikInverterBindingConstants.CHANNEL_POWER.equals(channelUID.getId())) {
-            if (command instanceof RefreshType) {
-                updateData();
-            }
-        }
+        // if (command instanceof RefreshType) {
```

Comment:
Is it necessary to handle `RefreshType`in the first place? It doesn't really seem wise to enable the general updateData() function (which updates all channels), for every request to Refresh a single channel.
Would is be enough to just rely on the existing scheduled update?

Reply:
I would say, it depends on the refresh frequency.
This REFRESH is mainly (but not only) used when you link a channel to an item. In mainUI, you expect to see a value more or less immediately.
So I would suggest to keep the handling of REFRESH command, except if you tell me that your refresh is scheduled very frequently.

Reply:
It's hardcoded to 10, for now at least.

Reply:
10 seconds? 
I suggest to keep the REFRESH handling.

Reply:
ok, done.